mantidproject/mantid
# Various documentation fixes

Description of work.
This pull request fixes various issues with the documentation such as links not working correctly or the text not being output due to syntax errors. As a side effect it should also reduce the number of documentation warnings to 1.

As a side note if anybody knows how to resolve this [issue](http://builds.mantidproject.org/job/master_doctests/381/warnings3Result/package.-42638972/file.-668951323/) it will bring syntax warnings down to 0
**To test:**
Check the number of syntax errors/warnings for the documentation build - there should only be 1. 

Fixes #17540 .



---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
